src/jit/flowgraph.cpp

```diff
@@ -4857,7 +4859,8 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, BYTE*
                     fgObserveInlineConstants(opcode, pushedStack, isInlining);
                 }
                 break;
-
+            case CEE_RET:
+                retBlocks++;
```

Comment:
Well out of my depth now... rename to `retBlocks`->`potentialRetBlocks` and include `CEE_TAILCALL`?